source/fundamentals/connection/connect.txt

```diff
@@ -17,8 +17,8 @@ Connection Guide
    :depth: 2
    :class: singlecol
 
-This guide shows you how to connect to a MongoDB instance or replica set
-using the Node.js driver.
+This guide shows you how to connect to a MongoDB Atlas instance, 
+another MongoDB instance, or a replica set using the Node.js driver.
```

Comment:
Suggestion:
I think it could be better to use "a MongoDB instance" given that "another" might not make sense in the context of the prior suggestion, which changes "instance" to "cluster".